src/lib/fetch-release-data.ts

```diff
@@ -74,7 +73,7 @@ export function getLatestVersionFromVersions(versions: string[]): string {
  */
 export function generateStaticProps(
   product: Product | string
-): Promise<GetStaticPropsResult<GeneratedProps>> {
+): Promise<{ props: GeneratedProps; revalidate: number }> {
```

Comment:
Here's the error, basically it gets mad when I try to access `props`:

<img width="1437" alt="CleanShot 2022-03-28 at 12 05 08@2x" src="https://user-images.githubusercontent.com/4624598/160440261-8f48ad2b-a40e-459a-83a7-e9af91c00c60.png">

Definitely happy to try to debug this and fix - can definitely see how it'd be ideal to use the type provided by next (in case that changes in the future or whatnot)

Reply:
Huh, weird! Well, let's roll with your fix for now 🙇  Thanks for the insight